src/configs/lang.ts

```diff
@@ -898,3 +898,17 @@ export const PLAN_CHANGES_HAVE_NOT_BEEN_SAVED = translate(
   'PLAN_CHANGES_HAVE_NOT_BEEN_SAVED',
   'Changes to plan have not been saved'
 );
+
+export const INVALID_GEOMETRIES = translate(
+  'INVALID_GEOMETRIES',
+  'Map failed to load: data has invalid geometries'
+);
+export const PLAN_NOT_FOUND = translate('PLAN_NOT_FOUND', 'Map failed to load: plan not found');
+export const JURISDICTION_NOT_FOUND = translate(
+  'JURISDICTION_NOT_FOUND',
+  'Map failed to load: Jurisdiction not found'
+);
+export const FOCUS_AREA_NOT_FOUND = translate(
+  'FOCUS_AREA_NOT_FOUND',
+  'Map failed to load: no focus area found'
+);
```

Comment:
What is the difference between these two?

Reply:
My bad, I think they both refer to same thing. will drop the focus area in favour of jurisdiction.

Reply:
on second thought since we use different slices for jurisdiction and focus areas if both fail we may have same error being flashed twice when we use the `JURISDICTION_NOT_FOUND` to represent both.

Reply:
Yeah I think they are the same thing to the web user